webpack/config.js

```diff
@@ -0,0 +1,19 @@
+console.log("current directory is " + __dirname);
+const path = require("path");
+
+module.exports = {
+  entry: "./webpack/app/foo.js",
```

Comment:
It seems this file doesn't exist. Should this point to a real file? Or, can this setting be removed?